components/Card/Card.tsx

```diff
@@ -0,0 +1,26 @@
+import Link from 'next/link';
+import * as S from './Card.Style';
+
+// data의 타입을 어떻게 정해줘야할지 의문 ...
+const Card = data => {
```

Comment:
주석 귀엽네요 ㅋㅋㅋ
우선, data라는 props를 받아주고 싶으면 이렇게 하셔야겠죠?

```suggestion
const Card = ({ data }) => {
```

다만, 이 과정에서 타입스크립트로 props type을 써준다고하면 interface를 만들어서 타입을 넣어주실 수 있어요 :)
```tsx
// response data인 { title, description, url, imageSource, createdAt } 에 해당되는 타입을 data롤 호출할때부터 미리 타입 정해주고, props type을 써줄때는 해당 타입을 import 받아옵니다. 

interface Props {
  data: Data
}

const Card = ({ data } : Props ) => {
...
```

Reply:
data의 타입을 정해줄 때, folder의 links 배열이 들어와야 하는 상황이라서 배열로 타입을 지정해줘야 한다고 생각했습니다. 그런데 'data: string[]'이라고 인터페이스 안에서 타입을 정해주면 map을 사용할 때 계속 에러가 생깁니다.. 그래서 타입을 어떻게 해줘야할지 도통 모르겠습니다ㅠㅠ